cmd/controller/main.go

```diff
@@ -54,7 +55,7 @@ func main() {
 		WithControllers(ctx, controllers.NewControllers(
 			ctx,
 			op.Manager,
-			op.Session,
+			awsConfig,
```

Comment:
```suggestion
			op.Config,
```

Can we just inline this?